packages/math-input/src/components/keypad/button.tsx

```diff
@@ -44,60 +49,76 @@ const styles = StyleSheet.create({
         boxSizing: "border-box",
         borderRadius: 7,
         border: "2px solid transparent",
-    },
-    outerBoxHover: {
-        border: `2px solid ${Color.blue}`,
-    },
-    outerBoxPressed: {
-        border: "2px solid #1B50B3",
-    },
-    clickable: {
-        width: "100%",
-        height: "100%",
-        boxSizing: "border-box",
+        ":hover": {
+            border: `2px solid ${Color.blue}`,
+        },
+        ":active": {
+            border: "2px solid #1B50B3",
+        },
     },
 });
 
-export type Props = {
+type OwnProps = {
     onPress: () => void;
     children: React.ReactNode;
     style?: StyleType;
     tintColor?: string;
+    keyConfig: KeyConfig;
+};
+
+type ReduxProps = {
+    gestureManager: GestureManager;
 };
 
-export default class Button extends React.Component<Props> {
+export type Props = OwnProps & ReduxProps;
+
+class Button extends React.Component<Props> {
     render(): React.ReactNode {
-        const {onPress, children, style, tintColor} = this.props;
+        const {children, style, tintColor, gestureManager, keyConfig} =
+            this.props;
+        const hovered = false;
+        const focused = false;
+        const pressed = false;
```

Comment:
These look unused now as they trigger use of styles that don't even exist anymore (such as `styles.outerBoxHover`)